pkg/vpa/service_test.go

```diff
@@ -141,151 +141,211 @@ func TestMakeAllVerticalContainerResourcePhaseRunning(t *testing.T) {
 	}
 }
 
-func TestService_UpdateVPAFromTortoiseRecommendation(t *testing.T) {
```

Comment:
This `TestService_UpdateVPAFromTortoiseRecommendation` is a completely new one, but removed after your rebase in your PR. Can you re-add it in the PR?
https://github.com/mercari/tortoise/blob/main/pkg/vpa/service_test.go#L144-L309